<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>431</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>710</width>
    <height>437</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-4</x>
     <y>-8</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/icons/img/Siemens_AISIN_Capital_.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/icons/img/service.png</normaloff>:/icons/img/service.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Update</string>
    </attribute>
    <widget class="QLabel" name="imglabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-2</y>
       <width>701</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/icons/img/Siemens_AISIN_Capital_.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="laterButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>310</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
font: 75 20pt &quot;Nirmala UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 4, 12);
border-radius:20px;
}

QPushButton:hover {
background-color: rgb(0, 15, 25);
}
</string>
     </property>
     <property name="text">
      <string>Later</string>
     </property>
    </widget>
    <widget class="QLabel" name="FOTA1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>18</y>
       <width>661</width>
       <height>86</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Palatino Linotype</family>
       <pointsize>48</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 48pt &quot;Palatino Linotype&quot;;
color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>FOTA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="textlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>661</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Welcome, here's a new update for your system.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="textlabel_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>661</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Do you want to update it now?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="updateButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QPushButton{
font: 75 20pt &quot;Nirmala UI&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 4, 12);
border-radius:20px;
}

QPushButton:hover {
background-color: rgb(0, 15, 25);
}
</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset resource="img.qrc">
      <normaloff>:/icons/img/idea.png</normaloff>:/icons/img/idea.png</iconset>
    </attribute>
    <attribute name="title">
     <string>History</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>2</y>
       <width>701</width>
       <height>401</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imglabel_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>701</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/icons/img/Siemens_AISIN_Capital_.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>691</width>
       <height>271</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>Update1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>Update5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Update details</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
